tide_logs.module

```diff
@@ -1,11 +1,19 @@
 <?php
 
+use \Drupal\Core\Site\Settings;
+
 /**
  * Implements hook_admin_audit_trail_log_alter().
  *
  * Responds to admin_audit_trail events and forwards it to tide_logs otel collector.
  */
 function tide_logs_admin_audit_trail_log_alter(&$log) {
+  // Only runs on production environment.
+  $environment = Settings::get('environment');
+  if ($environment !== 'production') {
+    return;
+  }
```

Comment:
This check added to avoid this issue for non prod environments where it spits out the notice in the build logs when enabling the tide_demo_content and ends up throwing php put of memory error.
i.g - 
![Screenshot 2023-10-17 at 3 45 06 pm](https://github.com/dpc-sdp/tide_logs/assets/20810541/1b9b6882-b8a9-4ce6-b3bd-f5b70aad46d7)

Reply:
As this was not the root cause of the memory out of limit issue, removing this production check from this hook.